src/components/analysis/AnalysisFiltersPanel.vue

```diff
@@ -33,6 +34,11 @@
       },
       onValueChange (field, value) {
         this.$emit('changeFilterValue', field, value)
+      },
+      applyFilters () {
```

Comment:
@yzhbankov 

AnalysisFilter panel reused in Look and in Explore, so it's not possible to dispatch action to explore store from here